src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -2117,6 +2119,25 @@ private IEnumerable<TypeInferenceInfo> InferTypeInVariableComponentAssignment(Ex
                 }
             }
 
+            private bool IsNullOrWhitespace(string text)
```

Comment:
It feels like such a helper method is likely to already exist. Can you double-check?